### ISSUE TITLE
Mobility scenario

### ISSUE THREAD

**Issue by gamalielpalomo** [CityScienceLab-GDL/cityscope-ccu#14]
We need to work on the mobility scenario:

- [ ] Create and compute the indicator (what variables are needed)
- [ ] Create the output format (heatmap, a number, etc).
- [ ] Create two comparative scenarios (one with the current state and another one with the intervention).

**Comment by gamalielpalomo** [comment]
Currently, we are planning to use the "Driving skill.gaml" mobility model (in gama, Plugin models/Driving Skill/Models/Driving skill.gaml) as a basis for our model. So @urbanistalepe is checking the required shape formats to correctly build the roads network.

**Comment by gamalielpalomo** [comment]
Thanks to @urbanistalepe, we had success in incorporating the mobility model to the cityscope model in a first version. It is still under development on branch dev-gama-may23. What's next is to check if everything is ok with the shape files and the model, and also create the mobility indicators such as: time spent in traffic by a person, total distance traveled, quantity of polution. We are intended to change polygon L1 to create a bridge and test its impact in the mobility when there is a cultural event (concert, etc) in the area.

**Comment by agrignard** [comment]
Ok that sounds great. Ideally I will let you merge it in the master branch?

**Comment by gamalielpalomo** [comment]
Yes, I will merge it in the master branch as soon as I am sure it is tested on the cityscope table
